src/components/ArticleModal.jsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Typography, Modal, CardMedia, IconButton } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+function ArticleModal({ onClose, articleId }) {
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://northcoders-news-be-s4h6.onrender.com/api/articles/${articleId}`
+      )
+      .then(({ data }) => {
+        console.log(data.article);
```

Comment:
remember to remove console.logs before merging into main to keep the console clear when others are running your project locally